hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/net/NetworkTopology.java

```diff
@@ -823,7 +823,7 @@ private static int getWeightUsingNetworkLocation(Node reader, Node node) {
           }
           currentLevel++;
         }
-        weight = (readerPathToken.length - currentLevel) +
+        weight = 2 + (readerPathToken.length - currentLevel) +
```

Comment:
nit: could we move "2 + " to the end of the next line, so that it is identical to what is in trunk? 

```
        weight = (readerPathToken.length - currentLevel) +
            (nodePathToken.length - currentLevel) + 2;
```